.github/workflows/build-reusable.yaml

```diff
@@ -67,6 +108,13 @@ jobs:
             -DinstallAtEnd=true \
             clean install
 
+      - name: Clean up Develocity
+        if: always()
```

Comment:
Why don't we simply remove the `if` condition instead?

Reply:
You are right, if the build fails, no need to clean it up.

Reply:
Fixed in https://github.com/apache/logging-parent/pull/229/commits/8975e12ce43007e9816e9996363e63e87c3ee477